src/pages/Account/Profile.tsx

```diff
@@ -114,7 +114,7 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
   return (
     <div className="Profile">
       <h2 className="login-welcome-title">
-        Hey, {githubHandle}. We’re so glad you’re here!
+        Hey, {firstName ?? "you"}. We’re so glad you’re here!
```

Comment:
Don't know if everyone is fine being called "Hey you".

Possible alternatives like Hello guest, friend whatever?

Reply:
The request came from @whoaitsaimz, the person who is responsible for making such product decisions.

![image](https://user-images.githubusercontent.com/63194656/124097554-5609ce00-da64-11eb-9eb4-7280b92bd5b8.png)